site/js/gipeda.js

```diff
@@ -434,10 +451,22 @@ function singleResult (res) {
     var name = res.name;
     var s = setting_for(name);
 
+    var value = res.value;
+    var unit = s.unit;
+
+    for (var i = 0; i < unitPrefix.length; i++) {
```

Comment:
Can you put this conversion code into its own function? It seems to be also called below in `compareResult`.